wps_shared/wps_shared/schemas/fire_watch.py

```diff
@@ -82,3 +81,28 @@ class FireWatchListResponse(BaseModel):
       
 class FireWatchFireCentresResponse(BaseModel):
     fire_centres: List[FireWatchFireCentre]
+
+
+class BurnForecastOutput(BaseModel):
+    id: int
+    fire_watch_id: int
+    date: int
+    temp: float
+    rh: float
+    wind_speed: float
+    ffmc: float
+    dmc: float
+    dc: float
+    isi: float
+    bui: float
+    hfi: float
+    in_prescription: str
+
+
+class FireWatchOutputBurnForecast(BaseModel):
+    fire_watch: FireWatchOutput
+    burn_forecasts: List[BurnForecastOutput]
+
+
+class FireWatchBurnForecastsResponse(BaseModel):
+    fire_watch_burn_forecasts: Dict[int, FireWatchOutputBurnForecast]
```

Comment:
What does the key in the dict represent here? Is it the unique `id` of a `FireWatch`? Is it possible to create a `FireWatch` schema with the `id` and a list of associated `FireWatchOutputBurnForecast` so marshalling on the frontend is simpler?

Reply:
Good idea! I'm on it.

Reply:
Simplified to return a list in https://github.com/bcgov/wps/pull/4493/commits/77c52106045d21ae9eda175ad9ca87ffad57eea4